lib/presentation/widgets/planned_expenses_list.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/material.dart';
+import 'package:trip_tally/presentation/theme/app_dimensions.dart';
+import 'package:trip_tally/presentation/widgets/planned_expenses.dart';
+
+import '../theme/app_paths.dart';
+
+class PlannedExpensesList extends StatelessWidget {
+  const PlannedExpensesList({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.all(AppDimensions.d30),
+      child: ListView.builder(
+        shrinkWrap: true,
```

Comment:
the use of shrinkwrap will make the list view to skip the lazy loading. Regarding expenses I think it's important to have the lazy loading available